docs/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,213 @@
+
+Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App Pay Kit SDK.
+
+{: .info }
+> When creating a checkout token, both `redirectConfirmUrl` and `redirectCancelUrl` must be set. Failing to do so will cause undefined behavior. The SDK’s example merchant server sets the parameters [here][example-server-props]{:target='_blank'}. See more by checking the [api reference][api-reference-props]{:target='_blank'}.
+
+## Step 1: Import the Cash App Pay Kit Dependency
+
+You can get the the latest version of the SDK from Maven. This is the import definition using Gradle:
+
+```gradle
+implementation "app.cash.paykit:core:1.0.3"
+```
+
+For definitions of other build systems, see [Cash App Pay Kit on Maven Central][cash-on-maven]{:target="_blank"}.
+
+{: .info }
+> Version `v1.0.3` of the SDK size is `12.3 kB`.
+
+## Step 2: Create a Cash App Pay Kit SDK Instance
+
+To create a new instance of the Cash App Pay Kit SDK, you must pass the `clientId`. This is a required field. This can be retrieved through the Afterpay object: `Afterpay.environment.payKitClientId`.
+
+{: .note }
+> Ensure that the Afterpay SDK is configured per the [instructions][configure-afterpay] before attempting to access `Afterpay.environment.payKitClientId`
+
+
+You should use `CashAppPayKitFactory` to create an instance of the Cash App Pay Kit SDK. When doing so, you'll need to specify the environment you will use, Sandbox or Production. The function `createSandbox()` will create an SDK instance in the Sandbox environment.
+
+{: .info }
+> You should use the Sandbox environment during the development phase and Production for your actual production releases.
+
+Creating a sandbox Cash App Pay Kit SDK instance:
+
+``` kotlin
+val payKit : CashAppPayKit = CashAppPayKitFactory.createSandbox(Afterpay.environment.payKitClientId)
+```
+
+Creating a production Cash App Pay Kit SDK instance:
+``` kotlin
+val payKit : CashAppPayKit = CashAppPayKitFactory.create(Afterpay.environment.payKitClientId)
+```
+
+{: .info }
+**Optional:** We recommend using the [Cash App Sandbox App][sandbox-app]{:target="_blank"} to test the payment flow in the Sandbox environment during your development phase.
+
+## Step 3: Register for State Updates
+
+To receive updates from Pay Kit, you’ll need to implement the `CashAppPayKitListener` interface. The interface exposes a single function, which gets called whenever there’s an internal state change emitted by the SDK:
+
+``` kotlin
+interface CashAppPayKitListener {
+   fun payKitStateDidChange(newState: PayKitState)
+}
+```
+
+You register with the SDK instance you’ve created above:
+
+``` kotlin
+payKit.registerForStateUpdates(this)
+```
+
+You should also use the unregister function when you're done with the SDK:
```

Comment:
```suggestion
You should also use the `unregister` function when you're done with the SDK:
```

Reply:
Addressed in https://github.com/afterpay/sdk-android/pull/185/commits/522aa7b27c98d5b78b5cc00a438de93adecf5e74

Again, register is not a function, so updated to bold text and capitalised.